src/components/common/topicsFilter/index.tsx

```diff
@@ -0,0 +1,265 @@
+  Collapse,
+  Select,
+  Radio,
+  Space,
+  Input,
+  DatePicker,
+} from "antd";
+import { LeftOutlined } from "@ant-design/icons";
+import { RootState } from "../../../store";
+import { useDispatch, useSelector } from "react-redux";
+import {
+  setIsReviewCanonizedTopics,
+  setFilterCanonizedTopics,
+} from "../../../store/slices/homePageSlice";
+
+import styles from "./createTopic.module.scss";
+
+const { Title, Text, Paragraph, Link } = Typography;
+const { Panel } = Collapse;
+const { Option } = Select;
+const mockDropdownList = ["blind_popularity", "mind_experts"];
+
+const infoContent = (
+  <>
+    <div className={styles.infoText}>
+      <Title level={5}>Score Value FIlter</Title>
+      <p>
+        Duis aute irure dolor in reprehderit in voluptate velit esse cillum
+        dolore eu fugiat nulla pariatur.
+      </p>
+    </div>
+  </>
+);
+
+const asContent = (
+  <>
+    <div className={styles.asfoText}>
+      <Title level={5}>Include review</Title>
+      <Paragraph>
+        Duis aute irure dolor in reprehderit in voluptalore eu fugiat nulla
+        pariatur.
+      </Paragraph>
+      <Title level={5}>Default</Title>
+      <Paragraph>
+        Duis aute irure dolor in reprehderit in voluptalore eu fugiat nulla
+        pariatur.
+      </Paragraph>
+      <Title level={5}>As of date</Title>
+      <Paragraph>
+        Duis aute irure dolor in reprehderit in voluptalore eu fugiat nulla
+        pariatur.
+      </Paragraph>
+    </div>
+  </>
+);
+
+function range(start, end) {
+  const result = [];
+  for (let i = start; i < end; i++) {
+    result.push(i);
+  }
+  return result;
+}
+
+function disabledDate(current) {
+  // Can not select days before today and today
+  return current && current < moment().endOf("day");
+}
+
+function disabledDateTime() {
+  return {
+    disabledHours: () => range(0, 24).splice(4, 20),
+    disabledMinutes: () => range(30, 60),
+    disabledSeconds: () => [55, 56],
+  };
+}
+
+const CreateTopic = () => {
+  const [isDatePicker, setIsDatePicker] = useState(false);
+  const [value, setValue] = useState(2);
+  const [datePickerValue, setDatePickerValue] = useState(null);
+  const [inputFilterValue, setInputFilterValue] = useState(0.0);
+  const dispatch = useDispatch();
+
+  const { algorithms } = useSelector((state: RootState) => ({
+    algorithms: state.homePage?.algorithms,
+  }));
+
+  const selectAlgorithm = (value) => {
+    dispatch(
+      setFilterCanonizedTopics({
+        algorithm: value,
+      })
+    );
+  };
+  const onChange = (e) => {
+    if (e.target.value === 3) {
+      setIsDatePicker(true);
+    } else {
+      setIsDatePicker(false);
+    }
+    setValue(e.target.value);
+  };
+  const pickDate = (e) => {
+    const IsoDateFormat = Date.parse(e?._d) / 1000;
+    setDatePickerValue(e?._d);
+    dispatch(
+      setFilterCanonizedTopics({
+        asofdate: IsoDateFormat,
+      })
+    );
+  };
+
+  const filterOnScore = (e) => {
+    const { value } = e.target;
```

Comment:
Change the variable name as it is already used before

Reply:
Fixed